src/sparseml/pytorch/optim/manager.py

```diff
@@ -26,7 +26,7 @@
 from torch.optim.optimizer import Optimizer
 
 from sparseml.optim import BaseManager, load_recipe_yaml_str, parse_recipe_variables
-from sparseml.pytorch.optim.modifier import Modifier, ScheduledModifier
+from sparseml.pytorch.sparsification.modifier import Modifier, ScheduledModifier
```

Comment:
can you check that this doesn't add any circular dependencies between optim and sparsification packages?